wordpress-mobile/WordPress-iOS
# Blog Details: Link to Calypso dashboard

Closes #5262 

To test:
View blog details for a wpcom user's blog.  Confirm you see a "Dashboard" option where "WP Admin" would otherwise appear.  Tap the Dashboard option and confirm that the wpcom calypso dashboard is loaded in safari (sign in if necessary). 
Confirm that the `WP Admin` continues to appear and function as intended for self-hosted blogs.

Needs review: @jleandroperez
